feature-flags/feature-flags.development.ts

```diff
@@ -6,11 +6,8 @@ export const flagsDevelopment: FeatureFlagsInterface = {
   FEATURE_TRANSPARENCY_NEW_TABLE: true,
   FEATURE_CU_INDEX_NEW_TABLE: true,
   FEATURE_CU_ABOUT_NEW_CONTAINER: true,
-  FEATURE_TRANSPARENCY_COMMENTS: true,
   FEATURE_GLOBAL_ACTIVITIES: true,
   FEATURE_AUTH: true,
-  FEATURE_MKR_VESTING: true,
-  FEATURE_AUDIT_REPORTS: false,
```

Comment:
Note that this is currently used to disable the Audit reports tab and we should keep this one in order to keep the tab disabled till it is necessary again

Reply:
ok, done